clidocstool_man_test.go

```diff
@@ -43,7 +43,7 @@ func TestGenManTree(t *testing.T) {
 	c, err := New(Options{
 		Root:      dockerCmd,
 		SourceDir: tmpdir,
-		Plugin:    true,
+		Plugin:    false,
```

Comment:
Why were these changes needed? (Just curious as I couldn't immediately associate with the change in this PR)

Reply:
Root commands for plugins are not generated; 

- https://github.com/docker/cli-docs-tool/blob/main/clidocstool_md.go#L50-L54
- https://github.com/docker/cli-docs-tool/blob/main/clidocstool_man.go#L61-L65
- https://github.com/docker/cli-docs-tool/blob/main/clidocstool_yaml.go#L110-L114